config/features.yml

```diff
@@ -12,6 +12,9 @@
 # Sorted using http://yaml-sorter.herokuapp.com/
 
 features:
+  sidekiq_benchmark_logging:
```

Comment:
Since the goal is to make this generic, I don't think having a single flag controlling duration logging across all the jobs that opt in - what if we want to turn it off for our jobs, but another team still wants theirs?

I think removing the flag entirely is the right option - logging for an extra day while we wait for a deploy by removing the inclusion line in the jobs we want to skip feels better than potentially stepping on another teams' logging.

Reply:
Makes sense - I've removed the flag